datafusion/sql/tests/sql_integration.rs

```diff
@@ -1225,22 +1225,6 @@ fn select_binary_expr_nested() {
     quick_test(sql, expected);
 }
 
-#[test]
-fn select_at_arrow_operator() {
```

Comment:
Should we add an explain test?

Reply:
I don't think it is needed since we don't really care about how it looks in plan.

Reply:
Maybe we can make sure `array_has_all` is applied 🤔